DoLahTest/ParserUnitTest.cpp

```diff
@@ -70,6 +90,31 @@ namespace DoLahTest {
         );
     }
 
+    TEST_METHOD(parseAddDetailedTest5) {
+        parseAddTestMethod((std::string)
+            "#cs2103 #homework on the #stage from 15th to 21st",
+            { "15/" + month + "/" + year + " ~ 21/" + month + "/" + year, "{ stage, homework, cs2103 }", "#cs2103 #homework on the #stage" }
```

Comment:
One area that in my consideration when writing test, is whether the test is easy or hard to maintain in the long run.